app/scripts/controllers/serviceInstance.js

```diff
@@ -204,6 +206,12 @@ angular.module('openshiftConsole')
         $scope.project = project;
         $scope.projectContext = context;
 
+        if ($scope.AEROGEAR_MOBILE_ENABLED) {
+          watches.push(MobileClientsService.watch(context, function(clients) {
+            $scope.mobileClients = clients.by("metadata.name");
```

Comment:
could be a constant "metadata.name" or some filter function created `listClientsByName()`

Reply:
This seems to be how they're doing it elsewhere, so I might just leave it as is for now